src/topology/algebra/const_mul_action.lean

```diff
@@ -348,8 +349,10 @@ export properly_discontinuous_smul (finite_disjoint_inter_image)
 
 export properly_discontinuous_vadd (finite_disjoint_inter_image)
 
-/-- The quotient map by a group action is open. -/
-@[to_additive "The quotient map by a group action is open."]
+/-- The quotient map by a group action is open, i.e. the quotient by a group action is an open
```

Comment:
Is "open quotient" standard terminology?

Reply:
I think it is, or at least I have already heard it to mean that, but a quick Google search makes me doubt... But I don't think that matters too much since the first part of the docstring is already very explicit

Reply:
We can maybe just keep the `The quotient map by a group action is open` part.

Reply:
@riccardobrasca I think it is used regularly, even if not completely standard, see
https://math.stackexchange.com/questions/4154061/why-is-the-open-quotient-of-a-second-countable-space-second-countable
https://math.stackexchange.com/questions/744493/open-quotient-of-second-countable-space-is-second-countable
I like the idea of keeping it in case someone is searching mathlib for the phrase "open quotient" in particular.  What do you think?